Cargo.toml

```diff
@@ -12,6 +12,20 @@ repository = "https://github.com/slog-rs/slog"
 readme = "README.md"
 autoexamples = true
 
+# This is our Minimum Supported Rust Version (MSRV)
+#
+# See the wiki for our full policy on changing this.
+#
+# There are two main requirements:
+# 1. Bumping the MSRV requires increasing the minor version number (2.7 -> 2.8)
+# 2. Changes must be clearly announced in the CHANGELOG.md file
+# 3. At a minimum, we must support at least the latest stable rust, minus two releases. 
```

Comment:
Seems to me we should constrain ourselves even more here.

Reply:
I'm just quoting the rules from the wiki here.

In practice, we've always supported much older versions than this, so it seems kind of odd that the wiki says this ;)

I see a few alternatives to this policy.
1. Just remove this explicit rule and make the whole thing informal.
2. Be much more strict, like l"atest stable rust minus 15 releases". Right now, this would guarentee (at a minimum) compatibility with 1.42 (released in March 2020)
    - We could always pick to support older versions, but we could not pick newer ones.
3. Explicitly tie our supported versions to the LTS releases of old linux distributions. For example, we could guarentee support for the rust versions in Debian OldStable (1.42), and CentOS 7 (1.37)
    - This is kind of of unpredictable, especially for people who don't use those linux distros

Option 2 seems the best. It's just a minor change that at this point would guarantee almost a year and a half of compatibility.

Rust releases follow a predictable schedule. There is a release every month, so this policy would reliably supports versions that are a year and a half old (in our case 1.42)

Reply:
Possibly the rules in the wiki were written when I was still improving things etc. . Nowadays the whole library seems mostly in slow maintenance, accumulated a lot of users and we should probably stick some a promise of at ~2 years of MSRV. "past 15 releases" sounds good to me.

Reply:
Okay, I have changed the version guarantee from "stable minus two" to "stable minus 15".

Re-open this conversation if there is still an issue :)